src/test/java/co/org/osso/api/ApliApplicationCalculatorTests.java

```diff
@@ -0,0 +1,60 @@
+package co.org.osso.api;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ApliApplicationCalculatorTests {
+
+    CalculatorService CalculatorServiceTarget = new CalculatorService();
+
+    @Test
+    public void checkCalculatorServiceFibonacci(){
+        List<Integer> fib = new ArrayList<>();
+        fib.add(1);
+        fib.add(1);
+        fib.add(2);
+        fib.add(3);
+        fib.add(5);
+        Assert.assertEquals(fib, CalculatorServiceTarget.getFibonacci(5));
+    }
+
+    @Test
+    public void checkCalculatorServiceFactorial(){
+        Assert.assertEquals(3628800, CalculatorServiceTarget.getFactorial(10));
```

Comment:
Pro tip (that I learned recently myself): you can put underscores (`_`) between the digits of large numbers to make it easier to read: `3_628_800`.

Reply:
Great thanks!